<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendPurEmail</class>
 <widget class="QWidget" name="ExtendPurEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidgetData">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>供应商</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>采购单号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>要求到货日</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>状态</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxAllSelect">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全选</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res.qrc">
           <normaloff>:/image/image/确认.png</normaloff>:/image/image/确认.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelEmail">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>邮箱：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditEmail">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExport">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string> 导出</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res.qrc">
           <normaloff>:/image/image/export.png</normaloff>:/image/image/export.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
